nengo/utils/progress.py

```diff
@@ -645,35 +647,32 @@ class MultiProgressTracker(object):
         The progress bar to display the progress.
     task : str
         Task name to display.
-    tasks : sequence of str
-        Subtask names to display.
     """
     def __init__(self, progress_bar, task):
-        self.progress_bar = wrap_with_progressupdater(
-            progress_bar=progress_bar)
-        self.task = task
-        self.total_progress = Progress(1, task=self.task)
-        self.subtask = None
-        self.subtask_progress = {}
+        super(MultiProgressTracker, self).__init__(1, progress_bar, task)
 
-    def activate_subtask(self, subtask):
-        self.subtask = subtask
-
-    def subprogress(self, max_steps):
+    def subprogress(self, max_steps, subtask):
         pt = ProgressTracker(
-            max_steps, self.progress_bar, self.subtask, close=False)
-        self.subtask_progress[self.subtask] = pt.progress
```

Comment:
I put this in do provide a way to retrieve the time of substeps (which can be useful information sometimes) ... though the instance is not exposed by the simulator tso there is no way to get access to this anyways at the moment.

Reply:
Upon further thought it doesn't really make sense to introduce this without a way to access it.